docs/dashboards/frequently-asked-questions.md

```diff
@@ -0,0 +1,108 @@
+FAQ
+===
+### How to connect store to the other components
+First you need to create the correct type of the store using correct constructor, and pass the data reference. More about this topic in the DataStore
```

Comment:
I would suggest to use `First, you need to create the correct type of store using the right constructor,`